src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        switch (new Random().nextInt(Shapes.values().length) + 1) {
+            case 1: return new Circle(
+                    new Random().nextDouble(100),
+                    ColorSupplier.getRandomColor());
+            case 2: return new Square(
+                    new Random().nextDouble(100),
```

Comment:
You create object of Random at least twice, so please define this new Random() on class level